dap-etl/extraction/src/main/scala/org/broadinstitute/monster/dap/common/OkWrapper.scala

```diff
@@ -26,7 +26,7 @@ class OkWrapper extends HttpWrapper {
         override def log(s: String): Unit = {
           logger.info(s)
         }
-      }).setLevel(Level.BASIC))
+      }).setLevel(Level.BODY))
```

Comment:
Was this change just for debugging? Would this affect anything? Perhaps storage size of logs?

Reply:
Updated to HEADERS